src/components/core/RadioGroup.js

```diff
@@ -8,33 +8,40 @@ export const RadioContext = React.createContext()
 
 const RadioGroup = ({
     value,
-    label,
-    helpText,
-    display,
+    label = '',
```

Comment:
Doesn't seem necessary to add defaults for `label` and `helpText`. The places where they are actually used (e.g. line 34 for `label`, it is checked before displaying the component in which it is used. Same with `helpText` https://github.com/dhis2/ui/blob/master/components/field/src/field/field.js#L38.

Also, the more I think about it, shouldn't `label` be a required prop? Radio buttons seem kind of useless without the label.